common/tools.py

```diff
@@ -1,22 +1,14 @@
-"""Collection of helper functions not fitting to other modules.
-"""
-# Back In Time
-# Copyright (C) 2008-2022 Oprea Dan, Bart de Koning, Richard Bailey,
-# Germar Reitze, Taylor Raack
```

Comment:
Should 'Germar Reitze' and 'Taylor Raack' still be credited in the new version of the file?

Reply:
See! That's why I need you! ❤️ 🤣 
Have missed that name because of the line break.